js/header.js

```diff
@@ -0,0 +1,34 @@
+import { switchMonth, findElemByDate, toggleSelectedDate } from "../month/months.js"
+import {switchWeekView} from "./weekView.js"
+import { selectedDate, setSelectedDate } from "./dateManipulation.js"
+
+const button_today = document.querySelector("#button_today")
+const headerControls_prev = document.querySelector("#headerControls_prev")
+const headerControls_next = document.querySelector("#headerControls_next")
+
+button_today.addEventListener('click', () => {
+    const today = new Date(Date.now())
```

Comment:
it's a tiny function, but you can probably create a function that returns today. You are already using it in a few places